pkgs/tools/backup/s3ql/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl, python3Packages, sqlite  }:
+
+python3Packages.buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "s3ql";
+  version = "2.13";
+
+  src = fetchurl {
+    url = "https://bitbucket.org/nikratio/${pname}/downloads/${name}.tar.bz2";
+    sha256 = "0bxps1iq0rv7bg2b8mys6zyjp912knm6zmafhid1jhsv3xyby4my";
+  };
+
+  propagatedBuildInputs = with python3Packages;
+    [ sqlite apsw pycrypto requests defusedxml dugong llfuse ];
+
+  meta = with stdenv.lib; {
+    description = "a full-featured file system for online data storage";
```

Comment:
@jagajaga Oops, my bad! Had just copy-pasted the description from their `setup.py`. Fixed now.